src/Poll.ts

```diff
@@ -117,6 +119,7 @@ export class Poll {
     }
 
     public vote(buttonText: string, userId: string): void {
+        if (this.isLocked) return;
```

Comment:
The user should be given some sort of message that there vote won't do anything if that's the route we're going

Reply:
Added